packages/mdc-button/_mixins.scss

```diff
@@ -124,6 +124,16 @@
     cursor: default;
     pointer-events: none;
   }
+
+  // stylelint-disable-next-line selector-max-type
+  svg {
+    position: relative;
+    top: 4px;
+    width: 18px;
+    height: 18px;
+    margin-right: 8px;
+    fill: currentColor;
+  }
```

Comment:
reenable stylelint after the svg selector

Reply:
This is `disable-stylelint-next-line`.